### ISSUE TITLE
Support Bessel polynomials?

### ISSUE THREAD

**Issue by MikaelSlevinsky** [MikaelSlevinsky/FastTransforms.jl#27]
And if so, what kind of fast transform is applicable? I've heard they're related to Jacobi polynomials with negative parameters, but they might transform nicely to Taylor polynomials.

**Comment by ajt60gaibb** [comment]
Do you want the Bessel-Fourier transform?

**Comment by MikaelSlevinsky** [comment]
Sure!

**Comment by marcusdavidwebb** [comment]
We discussed last year that the connection coefficients matrix between Bessel polynomials of different parameters is diagonally scaled Toeplitz-dot-Hankel, but decided in the end not to put it in the Fast Polynomials Transforms paper. Not sure if that Hankel matrix is positive definite or not.

See prop 5.1 of https://cmup.fc.up.pt/main/sites/default/files/publications/ConnectionCoefficients_CMUP_3.pdf <https://cmup.fc.up.pt/main/sites/default/files/publications/ConnectionCoefficients_CMUP_3.pdf>

Marcus

**Comment by MikaelSlevinsky** [comment]
Yeah it looks like the [reverse Bessel polynomials](https://en.wikipedia.org/wiki/Bessel_polynomials#Definition_as_a_hypergeometric_function) are related to generalized Laguerre polynomials whose orders are related to the degree... not sure which approximation space they're designed for.

**Comment by AshtonSBradley** [comment]
there is this for discrete Hankel
https://www.gnu.org/software/gsl/doc/html/dht.html 
but it would be great to have a Bessel-Fourier as part of FastTransforms.jl

**Comment by MikaelSlevinsky** [comment]
All contributions welcome! No reasonable pull request left behind.

Cheers,

Mikael

On Jun 10, 2018, at 9:25 AM, Alex Townsend <notifications@github.com<mailto:notifications@github.com>> wrote:


Back in September 2017, I was thinking about translating the transforms that I wrote in MATLAB:
https://github.com/ajt60gaibb/FastAsyTransforms
The algorithms in the GNU package are not optimal complexity and could be improved.
